ksqldb-parser/src/test/java/io/confluent/ksql/parser/properties/with/CreateSourcePropertiesTest.java

```diff
@@ -490,10 +492,26 @@ public void shouldGetKeyAndValueFormatFromFormat() {
             .build());
 
     // When / Then:
-    assertThat(props.getKeyFormat().get().getFormat(), is("KAFKA"));
+    assertThat(props.getKeyFormat(SourceName.of("foo")).get().getFormat(), is("KAFKA"));
     assertThat(props.getValueFormat().get().getFormat(), is("AVRO"));
   }
 
+  @Test
+  public void shouldGetKeyFormatFromAvroFormatWithAvroName() {
```

Comment:
nit:
```suggestion
  public void shouldSetAvroNameOnAvroKeyFormat() {
```
unless there's a separate test to verify the same behavior with `KEY_FORMAT='AVRO'` rather than `FORMAT='AVRO'`, I think that bit makes the name harder to grok rather than easier.